rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructure.java

```diff
@@ -337,14 +338,11 @@ public void configure(Object... parameters) {
             this.javaOptions = parameters[index++].toString();
 
             // credentials
-            if (parameters[index] == null) {
-                throw new IllegalArgumentException("Credentials must be specified");
-            }
             try {
                 persistedInfraVariables.put(CREDENTIALS_KEY,
                                             Credentials.getCredentialsBase64((byte[]) parameters[index++]));
             } catch (KeyException e) {
-                throw new IllegalArgumentException("Could not retrieve base64 credentials", e);
+                logger.debug("Provided credentials are not compatible with Base64 format.");
```

Comment:
The comment from @fviale also applies here. I  bet user should be invited to propose their own credential instead.